pkg/heatshrink/README.md

```diff
@@ -0,0 +1,9 @@
+# Introduction
+
+This package provides a compression library specifically developed for
```

Comment:
I think we should provide doxygen page for each package, but here is definitely the wrong place for that kind of discussion.